tests/conftest.py

```diff
@@ -743,25 +743,30 @@ def index_all_starred(empty_index, request):
 
 @pytest.fixture(
     params=[
-        {537286, 537287, 537288},
-        {537286},
-        {537287},
-        {537288},
-        {537286, 537287},
-        {537286, 537288},
-        {537287, 537288},
+        ({537286, 537288}, {537287}),
+        ({537286}, {537286}),
+        ({537287}, {537286, 537288}),
+        ({537288}, {537286, 537287, 537288}),
+        ({537286, 537287}, set()),
+        ({537286, 537288}, {537286, 537288}),
+        ({537287, 537288}, {537286, 537287, 537288}),
```

Comment:
If this is similar to the other mention/wildcard combination fixture, then we could consider extracting this as a fixture to be used in both places.

Reply:
I am not sure of the ideal place to refactor this, so I am creating an initial commit that extracts this as a fixture and the subsequent commits will just use this wherever needed. :)